src/app.py

```diff
@@ -696,6 +696,49 @@ def create_datastage():
         logger.error(e, exc_info=True)
         return Response("Unexpected error while creating a dataset: " + str(e) + "  Check the logs", 500)
 
+@app.route('/datasets/components', methods=['POST'])
+def multiple_components():
+    if not request.is_json:
+        return Response("json request required", 400)
+    entity_type = 'dataset'
+    try:
+        component_request = request.json
+        auth_helper = AuthHelper.configured_instance(app.config['APP_CLIENT_ID'], app.config['APP_CLIENT_SECRET'])
+        auth_tokens = auth_helper.getAuthorizationTokens(request.headers)
+        if isinstance(auth_tokens, Response):
+            return(auth_tokens)
+        elif isinstance(auth_tokens, str):
+            token = auth_tokens
+        elif 'nexus_token' in auth_tokens:
```

Comment:
The code is probably copied from the old `POST /datasets` function. As a matter of fact, `nexus_token` had been deprecated by Globus. Just remove this `elif` block here. We don't touch the old code though.